pkg/sensors/accelerator/device/device.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package device
+
+import (
+	"errors"
+	"time"
+
+	"golang.org/x/exp/maps"
+	"k8s.io/klog/v2"
+)
+
+var (
+	gpuDevices   = map[string]deviceStartupFunc{} // Static map of supported gpuDevices.
+	dummyDevices = map[string]deviceStartupFunc{} // Static map of supported dummyDevices.
+	qatDevices   = map[string]deviceStartupFunc{} // Static map of supported qatDevices.
+)
+
+type AcceleratorInterface interface {
+	// GetName returns the name of the device
+	GetName() string
```

Comment:
Here and everywhere else, don't use `Get` it should just be `Name() string`

Reply:
lol, Ok :)